src/styles.scss

```diff
@@ -10,3 +10,15 @@
   flex-wrap: wrap;
   gap: 1rem;
 }
+
+@media screen and (max-width: 768px) {
+  .iframe-video {
+    height: 250px;
+  }
+}
+
+.iframe-video {
+  border: 0px solid #000000;
+  background: #000000bf;
+  box-shadow: 0 0.25vmax 1vmax #000000bf;
+}
```

Comment:
I think we need to make the iframe responsive to work properly on mobile.

Reply:
You mean by using media-queries? I tried implementing dynamic resizing by changing height of iframe on window resize, but that didn't work well.

Reply:
You could use the media queries and breakpoints to ensure it fits the screen.
The current mobile view has scrollbars, i.e. on small devices like a phone:
![image](https://github.com/user-attachments/assets/90a1a2f7-5963-497e-a9e4-57402d1b7f93)

Reply:
That's only in the cases when the url is invalid. If I run it with valid url, it renders nicely:
<img width="724" alt="image" src="https://github.com/user-attachments/assets/ea5b9043-10a1-4922-a2e6-cafce5378b31">

Reply:
Oh I see, then it should be fine right away.